cdc/sink/codec/canal_test.go

```diff
@@ -147,7 +149,7 @@ func (s *canalEntrySuite) TestConvertEntry(c *check.C) {
 		},
 		Delete: true,
 		Columns: map[string]*model.Column{
-			"id": {Type: mysql.TypeLong, WhereHandle: &trueVar, Value: 1},
+			"id": {Type: mysql.TypeLong, WhereHandle: &trueVar, Value: 1, Flag: model.BinaryFlag},
```

Comment:
Why would you want to set the BinaryFlag on a long type?

Reply:
I just obey the information from `timodel.ColumnInfo.FieldType.Charset`, all type have their charset, and for `bigint` it shows the `Charset : binary` in my `cdc.log`.

Reply:
Sorry, it's my illusion.